### PR TITLE
Core: Do not include not shipped products to shipments (SH-107)

### PR DESCRIPTION
**Pull request by tulimaki**
Skip not shipped products when figuring out unshipped products.

Make default products that is created with `testing.factories.create_product`
shippable since it seems that all tests dealing with shipments until today
assumes that this is the case.

### CODE REVIEW

File: shuup/core/models/_orders.py [hunk 1]
```diff
@@ -824,6 +826,13 @@ def get_product_summary(self):
             .values_list("product_id", "quantity"))
         for product_id, quantity in lines:
             products[product_id]['ordered'] += quantity
+
+        from ._products import ShippingMode
```
Comment (Pikkupomo) [inline shuup/core/models/_orders.py]: Why is this import here?

File: shuup/core/models/_orders.py [hunk 2]
```diff
@@ -694,10 +694,12 @@ def create_shipment_of_all_products(self, supplier=None):
         :return: Saved, complete Shipment object
         :rtype: shuup.shop.models.Shipment
         """
+        from ._products import ShippingMode
```
Comment (Pikkupomo) [inline shuup/core/models/_orders.py]: Why is this import here?

File: doc/changelog.rst [hunk 3]
```diff
@@ -12,6 +12,7 @@ Unrealeased
 Core
 ~~~~
 
+- Do not include not shipped products to shipments
```
Comment (Pikkupomo) [inline doc/changelog.rst]: Changelog should also say that created products are `SHIPPED` by default now

File: shuup/core/models/_orders.py (hunk 2 above)
Comment (tulimaki) [inline shuup/core/models/_orders.py]: It can't be imported at the top of _orders since it causes circular import.

File: shuup/core/models/_orders.py (hunk 1 above)
Comment (tulimaki) [inline shuup/core/models/_orders.py]: It can't be imported at the top of _orders since it causes circular import. Probably shouldn't but couldn't figure the reason fast enough.
